src/app/pages/list/action-list/action-list.component.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActionListComponent } from './action-list.component';
+import { ActionListModule } from './action-list.module';
+
+describe('ActionListComponent', () => {
+    let component: ActionListComponent;
+    let fixture: ComponentFixture<ActionListComponent>;
+
+    beforeEach(async(() => {
+        void TestBed.configureTestingModule({
+            imports: [ActionListModule],
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(ActionListComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        void expect(component).toBeTruthy();
+    });
+
+    it('createItem', async(() => {
```

Comment:
Can you update these unit tests to read like actual sentences?

example:
it should [do what] [when]